### PR TITLE
Use FROM docs/base:latest again

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

File: docs/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,9 @@ RUN svn checkout https://github.com/kitematic/kitematic/trunk/docs /docs/content
 RUN svn checkout https://github.com/docker/tutorials/trunk/docs /docs/content/
 RUN svn checkout https://github.com/docker/docker/trunk/docs /docs/content/engine
 
+env PROJECT=opensource
```
Comment (thaJeztah) [inline docs/Dockerfile]: nit, but can you change this to uppercase `ENV`?

**Comment by thaJeztah** [comment]
One nit, but LGTM otherwise

**Comment by SvenDowideit** [comment]
de-nitted :)

**Comment by thaJeztah** [comment]
LGTM, thanks Sven :smile:

**Pull request merged**